decidim-core/lib/decidim/core/test/shared_examples/embed_resource_examples.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+shared_examples_for "an embed resource" do
```

Comment:
Can you use this in `decidim-assemblies` please?

Reply:
Actually, it seems we could add this to `decidim-participatory_processes` and `decidim-sortitions` too?